swan-lake/development-tutorials/source-code-dependencies/package-references.md

```diff
@@ -113,23 +112,78 @@ Ballerina strictly follows the rules of <a href="https://semver.org/" target="_b
 
 ### The `export` field
 
-You can specify the modules which should be visible to the outside using the `export` field. The export field accepts a string array, which contains the names of the modules which
-need to be publicly visible.
+By default, only the default module of the package is public. To control this behavior, specify the modules which should be visible to the outside using the `export` field under the `[package]` table. The export field accepts a string array, which accepts the names of the modules which need to be publicly visible. The modules should be referred in the `<pacakge_name>.<module_name>` format. The default module should be referenced by the package name.
+
+The following example shows how to specify the modules to expose:
 
-The modules should be referred in the `<pacakge_name>.<module_name>` format. The default module should be referenced by the package name.
+```toml
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+export = ["winery", "winery.model"] # The defaul module and the winery.model module will be public
+```
 
 ### The `icon` field
 
 The `icon` field accepts a path to an icon. The specified icon will be packaged into the `docs/` directory of the BALA.
 
 Only the `.png` format is supported for the icon.
 
+The following example shows how to specify an icon for the package.
+
+```toml
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+icon = "icon.png"
+```
+
 ### The `include` field
 
 You can provide paths to any additional resources, which need to be packed in the BALA file during the use of the `bal pack` command.
 
-The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory.
-The included file paths will be packaged into the root of the BALA preserving its original structure.
+The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The included file paths will be packaged into the root of the BALA preserving its original structure. 
+
+The paths should be relative to the package root directory and support the following patterns.
+
+```toml
+include = [
+    "foo", # any file/dir with name 'foo'
```

Comment:
```suggestion
    "foo", # Any file/dir named 'foo'.
```
Can we make the first letter capital in all the code comments and add a full stop at the end? And can we use this format for the rest also?